packages/manager/apps/dedicated/client/app/dedicated/server/installation/ovh/dedicated-server-installation-ovh.controller.js

```diff
@@ -315,7 +315,7 @@ angular
         const getHardRaid = $scope.getHardwareRaid();
         const getOvhTemplates = Server.getOvhTemplates($stateParams.productId)
           .then((templateList) => {
-            $scope.installation.desktopType = templateList.category;
+            $scope.installation.desktopType = templateList.category.sort();
```

Comment:
i think for sorting alphabetically iit's better to use :
```js
templateList.category.sort((a, b) => a.localeCompare(b))
```
as it handle accents better because
```
["e", "à", "é"].sort(); // ["e", "à", "é"]
["e", "à", "é"].sort((a, b) => a.localeCompare(b)); // ["à", "e", "é"]
```